manifests/cf-manifest/env-specific/prod-lon.yml

```diff
@@ -1,4 +1,4 @@
 ---
-cell_instances: 9
+cell_instances: 11
```

Comment:
Although it's fun to turn it up to 11, we really should go up to a multiple of 3 (ie 12) so that we have an equal number in each AZ.

Reply:
When AWS release a new AZ in London it might be ⅔ of an AZ like the last time :trollface:

Reply:
Follow up PR: https://github.com/alphagov/paas-cf/pull/1750